src/core/js/views/adaptView.js

```diff
@@ -18,6 +18,7 @@ class AdaptView extends Backbone.View {
     });
     this.isReact = (this.constructor.template || '').includes('.jsx');
     if (this.isReact) {
+      this.classSet = new Set(_.result(this, 'className').trim().split(/\s+/));
```

Comment:
Hi Tom,

Would you be able to make the instance property `this._classSet` private by prefixing it with an underscore please? 

It may also be prudent to change the names the variables throughout to give future readers a better indication as to their function?

`this.classSet`, `const classes` and `const difference` seems somewhat abstract. Could I perhaps suggest something along the lines of `this._renderedClasses`, `const currentClasses`, `const deleteClasses` or similar?

I look forward to hearing from you.

Warm regards,

Ollie.